src/components/manager/ManagerIngestionContent.js

```diff
@@ -13,233 +13,155 @@ import {
+};
+
+
+const ManagerIngestionContent = () => {
+    const dispatch = useDispatch();
+    const history = useHistory();
+    const servicesByID = useSelector(state => state.services.itemsByID);
+
+    return <RunSetupWizard
+        workflowSelection={({workflowSelectionValues, setWorkflowSelectionValues, handleWorkflowClick}) => (
+            <IngestWorkflowSelection
+                values={workflowSelectionValues}
+                setValues={setWorkflowSelectionValues}
+                handleWorkflowClick={handleWorkflowClick}
+            />
+        )}
+        workflowSelectionTitle="Table & Workflow"
+        workflowSelectionDescription={
+            <span style={{letterSpacing: "-0.1px"}}>Choose a table and ingestion workflow.</span>
         }
-    }
-
-    render() {
-        return <Layout>
-            <Layout.Content style={LAYOUT_CONTENT_STYLE}>
-                <Steps current={this.state.step} onChange={this.handleStepChange}>
-                    <Steps.Step title="Table & Workflow"
-                                description={<span style={{letterSpacing: "-0.1px"}}>
-                                    Choose a table and ingestion workflow.
-                                </span>}>
-
-                    </Steps.Step>
-                    <Steps.Step title="Input"
-                                description="Select input data for the workflow."
-                                disabled={this.state.step < STEP_INPUT &&
-                                    Object.keys(this.state.inputs).length === 0} />
-                    <Steps.Step title="Run" description="Confirm details and run the workflow."
-                                disabled={this.state.step < STEP_CONFIRM && (this.state.selectedWorkflow === null ||
-                                    Object.keys(this.state.inputs).length === 0)} />
-                </Steps>
-                <div style={{marginTop: "16px"}}>{this.getStepContents()}</div>
-            </Layout.Content>
-        </Layout>;
-    }
-}
-
-ManagerIngestionContent.propTypes = {
-    ...dropBoxTreeStateToPropsMixinPropTypes,
-    ...workflowsStateToPropsMixinPropTypes,
-    servicesByID: PropTypes.object, // TODO: Shape
-    projectsByID: PropTypes.object,  // TODO: Shape
-    tablesByServiceID: PropTypes.object,  // TODO: Shape
-    isSubmittingIngestionRun: PropTypes.bool,
+        confirmDisplay={({selectedWorkflow, workflowSelectionValues, inputs, handleRunWorkflow}) => (
+            <IngestConfirmDisplay
+                selectedTable={workflowSelectionValues.selectedTable}
+                selectedWorkflow={selectedWorkflow}
+                inputs={inputs}
+                handleRunWorkflow={handleRunWorkflow}
+            />
+        )}
+        onSubmit={({workflowSelectionValues, selectedWorkflow, inputs}) => {
+            const {selectedTable} = workflowSelectionValues;
+
+            if (!selectedTable || !selectedWorkflow) {
+                // TODO: GUI error message
```

Comment:
Ok if not in this version? Later we could implement an error display system with redux, so we can easilly show GUI error messages by using dispatch.

Reply:
there's one built into ant design, i'll just add one now

Reply:
done